### DISCUSSION TITLE
[Feature Request] Dirtywave M8 Music Tracker Headless

### DISCUSSION THREAD

**Discussion by rekleiner** [OnionUI/Onion#1492] [Ideas]
Please excuse my non programming background and porting ignorance. I am so ignorant I don't even know if this is possible. I saw this awesome music software modelled after the old Gameboy music tracker synthesizer software Little Sound DJ, recently ported to other platform.  This music tracker looks to be of high quality and fun.  I think it would be an awesome addition to OnionOS. I don't even know if its possible or if similar architecture. I know its running on a "Teensy" in the commercial version.

The project has a github and is seems to be based on a reknown commercial product. I will try to link some important pages below:

[Dirtywave M8 Tracker product page](https://dirtywave.com/products/m8-tracker-model-02)
[M8 Firmware GitHub](https://github.com/Dirtywave/M8Firmware)
[M8 Headless Firmware GitHub](https://github.com/Dirtywave/M8HeadlessFirmware)
[M8 Display Touch designer GitHub](https://github.com/DirtyWave/M8DisplayTouchdesigner)

Examples:
[Dirtywave M8 Tracker Headless running on R36S (RG351 RGB20 RK3326 clone)](https://youtu.be/tt95ZG6W8hM?si=Sx8BHpW4fib_cTDx)
[M8 Headless on the Anbernic RG353VS](https://youtu.be/0svPgtA53uQ?si=_1SZ1I2e94r8u5jO)
[DIY Dirtywave M8 headless using a Teensy 4.1 microcomputer](https://youtu.be/gf1Blf8M7XE?si=tUMqqpgYZUvYXwDY)

I apologize if this is the wrong place for requests of this nature.

Thank you for your time and consideration.

**Comment by Nighted** [comment]
There is this one: https://github.com/djdiskmachine/LittleGPTracker

Not what you're looking for but there is a tracker for Miyoo.

**Reply by rekleiner** [reply]
Good suggestion.  This one however seems much less capable. M8 Tracker is used by pros all around the world. Worth trying LittleGPTracker out though. 

Thank you.

**Comment by Nighted** [comment]
> M8 Tracker is used by pros all around the world. Worth trying LittleGPTracker out though.
> 
> Thank you.

Weird, never heard of it. I always used OpenMPT. ¯\_(ツ)_/¯

I haven't tried LGPT yet, haven't put it on the Miyoo yet. I spent way more time tweaking than I had been playing games so now I'm working on becoming a professional Pac-Man player. ;)
